### ISSUE TITLE
Chained call on next line in arrow function

### ISSUE THREAD

**Issue by ewired** [DanielXMoore/Civet#367]
I call .filter() on a variable in an arrow function with an added level of indentation under it:
```coffee
useEffect :void =>
  criteria.value = debouncedWorkingCriteria
    .filter (_, i) => validCriteria.value.has(i)
```
Civet (playground) interprets this as calling .filter() after the useEffect call:
```ts
useEffect((): void => {
  criteria.value = debouncedWorkingCriteria;
}).filter((_, i) => validCriteria.value.has(i));
```
I expected it to be within the anonymous function like this:
```ts
useEffect((): void => {
  criteria.value = debouncedWorkingCriteria
    .filter((_, i) => validCriteria.value.has(i));
});
```

**Comment by edemaine** [comment]
Minimal reproductions:

```coffee
useEffect =>
  x
  .y
```

```coffee
useEffect =>
  x
    .y
```

I figured out why this is happening: the implicit function call to `useEffect` turns off trailing member properties, and we don't turn them back on within arrow functions (or maybe blocks in general).

Luckily we're in a good position to solve this now, via #356. I'm working on a fix now.

**Comment by edemaine** [comment]
Should be fixed in #368, but I'd like to wait for review (likely later today) before releasing.

**Comment by edemaine** [comment]
Released in 0.5.71. Let us know if you have any other issues!
